Header header
raw_data/CameraRaw[] data